amity.py

```diff
@@ -0,0 +1,285 @@
+import os
+import random
+
+from tabulate import tabulate
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from db.migration import Base, Person, Room, DatabaseCreator
+from people.people import Fellow, Staff
+from rooms.rooms import LivingSpace, Office
+
+
+class Amity(object):
+    """This class holds the system's core functionality"""
+    rooms_list = {}
+    people_list = {}
```

Comment:
if these two were lists of instances of the rooms and the people then your work would be easier.